Eclipse installation is failing

See: https://github.com/rebornplusplus/contest-iso/actions/runs/11423696240/job/31783199296#step:4:42177


```
+ echo -e '\nExecuting eclipse.sh ...\n'
+ bash scripts/eclipse.sh
+ ECLIPSE_VERSION=2023-12
+ wget -c -O eclipse.tar.gz https://mirror.kakao.com/eclipse/technology/epp/downloads/release/2023-12/R/eclipse-java-2023-12-R-linux-gtk-x86_64.tar.gz
--2024-10-20 05:23:22--  https://mirror.kakao.com/eclipse/technology/epp/downloads/release/2023-12/R/eclipse-java-2023-12-R-linux-gtk-x86_64.tar.gz
Resolving mirror.kakao.com (mirror.kakao.com)... 113.29.189.165
Connecting to mirror.kakao.com (mirror.kakao.com)|113.29.189.165|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-10-20 05:23:23 ERROR 404: Not Found.

Error: Process completed with exit code 8.
```

--- a/scripts/eclipse.sh
+++ b/scripts/eclipse.sh
@@ -2,7 +2,7 @@
 
 set -eux
 
-ECLIPSE_VERSION="2023-12"
+ECLIPSE_VERSION="2024-09"
 
 # download eclipse: https://www.eclipse.org/downloads/packages/
 wget -c -O "eclipse.tar.gz" \

